spec/models/doc_sentencing_county_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe DocSentencingCounty, type: :model do
+  describe 'validations' do
```

Comment:
Don't forget to add in the association check for `belongs_to :county, optional: true`

Reply:
added in!